paddle/phi/api/yaml/backward.yaml

```
@@ -1835,6 +1835,12 @@
     data_type : x
   optional : boxes_num
 
+- backward_op : pull_sparse_v2_grad
+  forward : pull_sparse_v2 (Tensor[] ids, Tensor[] w, int embedding_dim = 11, int table_id = 0, str accessor_class = "", str ctrlabel_name = "", int padding_id = 0, bool scale_sparse_grad = true, str[] input_names = {}, bool is_distributed = true) -> Tensor[](out)
+  args : (Tensor[] ids, Tensor[] w, Tensor[] out_grad, int embedding_dim, int table_id, str accessor_class, str ctrlabel_name, int padding_id, bool scale_sparse_grad, str[] input_names, bool is_distributed)
```

```suggestion
  args : (Tensor[] ids, Tensor[] w, Tensor[] out_grad, int embedding_dim = 11, int table_id = 0, str accessor_class = "", str ctrlabel_name = "", int padding_id = 0, bool scale_sparse_grad = true, str[] input_names = {}, bool is_distributed = true)
```
现在的报错是这里造成的，需要设置一下默认值，否则python脚本就用None填充了。

---

感谢～已修改